compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/create_logical_objectfifo_link.mlir

```diff
@@ -37,6 +37,40 @@ func.func @link_multiple_inputs(%arg0: memref<32x1024xi32>, %arg1: memref<32x64x
 
 // -----
 
+// CHECK-LABEL: func.func @link_multiple_inputs_with_offsets
+// CHECK:       %[[DMA0:.+]] = amdaie.circular_dma_cpy_nd
+// CHECK:       %[[DMA1:.+]] = amdaie.circular_dma_cpy_nd
+// CHECK:       %[[DMA2:.+]] = amdaie.circular_dma_cpy_nd
+// CHECK:       %[[DMA3:.+]] = amdaie.circular_dma_cpy_nd
+// CHECK:       amdaie.logicalobjectfifo.link[%[[DMA0]], %[[DMA2]], %[[DMA1]]] -> [%[[DMA3]]] ()
```

Comment:
Add a comment explaining why you expect DMA2 to appear before DMA1 (because it has a lower overall offset).

Reply:
Done